### DISCUSSION TITLE
Firefox | Fronzen columns

### DISCUSSION THREAD

**Discussion by ilmandone** [Comcast/react-data-grid#2585] [Q&A]
On FF 89 / FFD 90.0b7 frozen columns not working properly.

It seems a problem of z-index.
The issue is not present into header's cell.

**Comment by nstepien** [comment]
Are you on the latest canary version? I can't reproduce the issue here.
https://adazzle.github.io/react-data-grid/canary/?path=/story/demos--common-features
![image](https://user-images.githubusercontent.com/567105/121877066-2176e200-cd02-11eb-9654-7ab8486e12ba.png)

**Comment by ilmandone** [comment]
Yes: last canary version .49 .

I found the origin of this issue.
It seems that only on FF if you apply a pseudo element on a cell at mouse hover the frozen feat glitch.

On FF
![image](https://user-images.githubusercontent.com/7240510/121891427-b508dc80-cd1b-11eb-8854-963c36820a73.png)

On Chrome
![image](https://user-images.githubusercontent.com/7240510/121891509-cf42ba80-cd1b-11eb-8ca1-ea62761a7997.png)


CSS code:
```
.rdg .rdg-row:hover .rdg-cell:after{
    content: "";
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    display: block;
    background-color:rgba(0,0,0,0.1);    

    pointer-events: none;
}
```

**Comment by ilmandone** [comment]
Add:

Same pseudo element on the row works fine.
